junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileArgumentsProvider.java

```diff
@@ -105,7 +105,7 @@ public boolean hasNext() {
 
 		@Override
 		public Arguments next() {
-			Arguments result = arguments(this.nextCsvRecord);
+			Arguments result = of(this.nextCsvRecord);
```

Comment:
It won't because you can't statically import two methods of the same name. :wink:

I can still change it but I prefer it this way, precisely because the return type is at the beginning of the line.